README.md

```diff
@@ -61,7 +61,7 @@ To setup an admin user on mongodb:
 
     Enter the mongo shell
     use admin
-    db.addUser( { user: "admin", pwd: "password", roles: [ "clusterAdmin", "readWriteAnyDatabase", "userAdminAnyDatabase", "dbAdminAnyDatabase"] } )
+    db.createUser( { user: "admin", pwd: "password", roles: [ "clusterAdmin", "readWriteAnyDatabase", "userAdminAnyDatabase", "dbAdminAnyDatabase"] } )
```

Comment:
I have the current stable release 3.0.3 which is likely the issue.  I'll install 2.4 for now.